packages/web/src/pages/private-key-exporter-page/PrivateKeyExporterPage.module.css

```diff
@@ -0,0 +1,37 @@
+.horizontalLogo {
+  cursor: pointer;
+  max-width: 132px;
+}
+
+.backToSettings {
+  cursor: pointer;
+}
+
+.backToSettings p {
+  transition: all 0.07s ease-in-out;
+  transform: scale3d(1, 1, 1);
+}
+
+.backToSettings:hover p {
+  transform: scale3d(1.03, 1.03, 1.03);
+}
+
+.backToSettings:active p {
+  transform: scale3d(0.97, 0.97, 0.97);
+}
```

Comment:
Is back to settings not just a plain button? Wondering why we need the custom styling instead of using a harmony component.

Reply:
using harmony now